psi4/src/psi4/libsapt_solver/fdds_disp.cc

```diff
@@ -156,22 +157,81 @@ FDDS_Dispersion::FDDS_Dispersion(std::shared_ptr<BasisSet> primary, std::shared_
     dfh_ = std::make_shared<DFHelper>(primary_, auxiliary_);
     dfh_->set_memory(doubles);
     dfh_->set_method("DIRECT_iaQ");
```

Comment:
`if (is_hybrid) {...} else {...}`

Reply:
Everything outside the if statement is common to both is_hybrid=True and is_hybrid=False. The code should be self-consistent for both case.